cli/src/merge_tools/diff_working_copies.rs

```diff
@@ -132,7 +132,7 @@ pub(crate) fn check_out_trees(
 ) -> Result<DiffWorkingCopies, DiffCheckoutError> {
     let changed_files: Vec<_> = left_tree
         .diff_stream(right_tree, matcher)
-        .map(|(path, _diff)| path)
+        .map(|TreeDiffEntry { target, .. }| target)
```

Comment:
we don't need to since the `CopyRecords` are constructed directly here and guaranteed to be empty.  We would have to pass them in from an external source to make it work here at which point we are unlikely to forget anything since we are so close to this line.